plugins/GrapesJsBuilderBundle/package.json

```diff
@@ -26,20 +40,34 @@
   },
   "homepage": "https://github.com/mautic/plugin-grapesjs-builder#readme",
   "dependencies": {
-    "grapesjs": "^0.17.29",
-    "grapesjs-mjml": "^0.5.8",
+    "grapesjs": "^0.21.8",
```

Comment:
there are some breaking changes that we are affected by. Are you aware of that: https://github.com/GrapesJS/grapesjs/releases/tag/v0.20.1

Reply:
@adiux Yes, I know. Those changes have been addressed by changes in the other files (eg. builder service and the grapes js preset mautic dependency.